wagtailnhsukfrontend/mixins.py

```diff
@@ -23,3 +26,40 @@ class ReviewDateMixin(models.Model):
     class Meta:
 
         abstract = True
+
+
+class HeroMixin(models.Model):
+    """
+    A Mixin to add the model for Hero components.
+
+    """
+
+    toggle_hero_text = models.BooleanField(default=True)
+    toggle_hero_image = models.BooleanField(default=False)
+    hero_text = models.TextField(blank=True, default='')
+    hero_heading = models.TextField(default='')
+    hero_image = models.ForeignKey(
+        Image,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+',
+    )
+
+    hero_contents = MultiFieldPanel([
```

Comment:
I think the convention here is:
```content_panels = [
  MultiFieldPanel([...])
]
```
Because then the page model becomes
```
content_panels = Page.content_panels + HeroMixin.content_panels + SomeOtherMixin.content_panels + [...]
```

Reply:
Ahh, I only did it this way as i thought it would trigger flake8 otherwise?